src/ui/components/SignTransactionButton/SignTransactionButton.tsx

```diff
@@ -85,15 +90,12 @@ export const SignTransactionButton = React.forwardRef(
         onClick={onClick}
         render={({ handleClick }) => (
           <Button
-            disabled={sendTxMutation.isLoading}
+            disabled={isLoading}
```

Comment:
Can there be a problem with disabled option being overwritten by `buttonProps.disabled = false` ?

Reply:
Hm, good catch, I'll add comment